metrics/resource_view.go

```diff
@@ -19,6 +19,7 @@ package metrics
 import (
 	"errors"
 	"fmt"
+	"log"
```

Comment:
The `log` packages produces a different (unstructured) log format -- I'd try to either wire a logger in somehow (if necessary) or drop the use of `log` here.

Reply:
Done.